osu.Game/Screens/Select/Footer.cs

```diff
@@ -0,0 +1,115 @@
+﻿//Copyright (c) 2007-2016 ppy Pty Ltd <contact@ppy.sh>.
+//Licensed under the MIT Licence - https://raw.githubusercontent.com/ppy/osu/master/LICENCE
+
+using System;
+using OpenTK;
+using OpenTK.Graphics;
+using osu.Framework.Graphics;
+using osu.Framework.Graphics.Containers;
+using osu.Framework.Graphics.Sprites;
+using osu.Framework.Graphics.Transformations;
+using osu.Game.Graphics;
+using osu.Game.Graphics.UserInterface;
+using osu.Game.Screens.Menu;
+
+namespace osu.Game.Screens.Select
+{
+    public class Footer : Container
+    {
+        private Box modeLight;
+
+        private const float play_song_select_button_width = 100;
+        private const float play_song_select_button_height = 50;
+
+        public const int TRANSITION_LENGTH = 300;
+
+        private const float padding = 80;
+
+        public Action OnBack;
+        public Action OnStart;
+
+        private FlowContainer buttons;
+
+        public void AddButton(string text, Color4 colour, Action action)
+        {
+            var button = new FooterButton
+            {
+                Text = text,
+                Height = play_song_select_button_height,
+                Width = play_song_select_button_width,
+                SelectedColour = colour,
+                DeselectedColour = colour.Opacity(0.5f),
+            };
+
+            button.Hovered = () => updateModeLight(button);
+            button.HoverLost = () => updateModeLight();
+            button.Action = action;
+            buttons.Add(button);
+        }
+
+        private void updateModeLight(FooterButton button = null)
+        {
+            modeLight.FadeColour(button?.SelectedColour ?? Color4.Transparent, TRANSITION_LENGTH, EasingTypes.OutQuint);
+        }
+
+        public Footer()
+        {
+            const float bottomToolHeight = 50;
+
+            RelativeSizeAxes = Axes.X;
+            Height = bottomToolHeight;
+            Anchor = Anchor.BottomCentre;
+            Origin = Anchor.BottomCentre;
+            Children = new Drawable[]
+            {
+                new Box
+                {
+                    RelativeSizeAxes = Axes.Both,
+                    Size = Vector2.One,
+                    Colour = Color4.Black.Opacity(0.5f),
+                },
+                modeLight = new Box
+                {
+                    RelativeSizeAxes = Axes.X,
+                    Height = 3,
+                    Position = new Vector2(0, -3),
+                },
+                new OsuLogo()
+                {
+                    Anchor = Anchor.BottomRight,
+                    Scale = new Vector2(0.4f),
+                    Position = new Vector2(-70, -25),
+                    Action = () => OnStart?.Invoke()
+                },

```

Comment:
i don't mind having it as-is right now. this is a temporary solution as i believe @arflyte is looking to make a button here rather than having the logo.